app/views/api/notes/_note.rss.builder

```diff
@@ -19,4 +19,8 @@ xml.item do
   xml.geo :lat, note.lat
   xml.geo :long, note.lon
   xml.georss :point, "#{note.lat} #{note.lon}"
+
+  note.tags.each do |k, v|
+    xml.tag(:k => k, :v => v)
+  end
```

Comment:
Is it going to be understood by any rss readers?

Reply:
Thanks for catching this. Honestly, I'm not sure about what could be the best solution here. I believe we have multiple options:

1. Adding Atom + XHTML namespaces and displaying tags like in [user's history feed](https://www.openstreetmap.org/user/Nenad%20Vujicic/history/feed) (I'm a little bit afraid to add new namespaces because of client support)
2. Using e.g. <dc:subject> for simplified, flattened representation of tags (the easiest and the most portable, but not so nice visual representation of tags)
3. Creating custom or some other namespace or some other solution?

Can you suggest what could be the best approach here?

Reply:
Or perhaps using one dc:subject element per note-tag pair (supported by the DCMI standard)? This was used in the most recent push.

Reply:
Or not writing the tags in rss for now. Currently nobody is looking for them there.

Reply:
Thanks, removed!